Bluepill-cli/Bluepill-cli/Bluepill/Bluepill.m

```diff
@@ -425,6 +489,21 @@ - (void)deleteSimulatorWithContext:(BPExecutionContext *)context andStatus:(BPEx
     [context.runner deleteSimulatorWithCompletion:handler.defaultHandlerBlock];
 }
 
+//only called when bp is running in the delete only mode
+- (void)deleteSimulatorOnlyTaskWithContext:(BPExecutionContext *)context {
+    
+    if ([context.runner useSimulatorWithDeviceID: [[NSUUID alloc] initWithUUIDString:context.config.deleteSimUDID]]) {
```

Comment:
When we are talking about that long string, I'd rather we call it UDID like Apple does.

Maybe we should talk about `Sim` or `Simulator` in Bluepill in general _except_ when directly referring to the Private Frameworks, in which case it should be `Device`. Kind of like a hint about what is in private frameworks and what we have control over. So in this case, `useSimulatorWithUDID` would be more appropriate. Inside the private frameworks we can talk about `deviceUDID`.